src/SegmentedButton/Segment.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+import * as React from 'react';
+import { View, TouchableWithoutFeedback } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { Text } from '../Text';
+import { StyleSheet } from '../PlatformStyleSheet';
+
+export type Props = {|
+  +label: string,
+  +value: string,
+  +first: boolean,
+  +last: boolean,
+  +active: boolean,
+  +onPress: (value: string) => void,
+|};
+
+export default class Segment extends React.Component<Props> {
+  handleOnPress = () => {
+    const { value, onPress } = this.props;
+    onPress(value);
+  };
+
+  render() {
+    const { label, first, last, active } = this.props;
+    return (
+      <TouchableWithoutFeedback onPress={this.handleOnPress}>
+        <View
+          style={[
+            styles.container,
+            first && styles.containerFirst,
```

Comment:
As I refactored `Text` in a previous PR, maybe it would be cleaner to remove all these undefined from `style` and creating a helper function.

Reply:
I can definitely do that. But is there any added value in it, besides slightly nicer test snapshots? For me it's quite similar and we are using current `[styles.label, active && styles.labelActive]` pattern in multiple components. It is also mentioned in StyleSheet part of RN docs ( https://facebook.github.io/react-native/docs/stylesheet ).